addons/mail/static/src/new/discuss/discuss.js

```diff
@@ -98,6 +98,17 @@ export class Discuss extends Component {
         return this.store.threads[this.store.discuss.threadLocalId];
     }
 
+    get hasDiscussContent() {
+        return (
+            this.thread &&
+            !(
+                this.store.isSmall &&
+                (this.store.discuss.activeTab === "channel" ||
+                    this.store.discuss.activeTab === "chat")
+            )
+        );
+    }
```

Comment:
I've slightly simplified the expression, and immediately dump it in the template (no need to define a `getter`)